src/localization/src/main.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+"""Script to initialize tree tf
+
+@author: Jorge Muñoz    
+@date: 20221125
+"""
+
+import rospy
+from handle import Localization
+
+def main():
+    rospy.init_node('car_broadcaster', anonymous=True)
+    Localization()
+    rospy.spin()
+    rospy.logwarn("spin")
```

Comment:
`logwarn` after `rospy.spin()` won't run until the latter returns, so it doesn't make much sense nor is it required.